network/horizon/admin-guide/error-handling.mdx

```diff
@@ -0,0 +1,182 @@
+
+There are some resolution strategies that are common between the 2 endpoints while some strategies are more endpoint specific.
+
+### Request Adjustments
+
+Certain transaction submission failures also need adjustments to succeed.
+
+- If the XDR is malformed, or the transaction is otherwise invalid, you’ll encounter a `400 Bad Request` (for example, see excluding a source account). Both transactions and their operations can be easily malformed: look at the `extras.result_codes` field for details and cross-reference them with the appropriate result codes documentation to determine specifics.
+- Transaction fees are also a safe adjustment. If you get a `tx_insufficient_fee` error, refer to the Insufficient Fees and Surge Pricing section later in this document.
+
+### Polling and Retrying Transactions
+
+#### Async Transaction Submission
+
+Submissions using the async endpoint return a clear response back from core. There are different actions that clients can take based on the specific `tx_status` returned:
+
+1. `PENDING`: the submission is successful but the txn is still waiting to be included in the ledger. You should use the `GET /transactions/:hash` endpoint to poll for the submitted txn and check if it makes into the ledger.
+2. `ERROR`: the submission did not go through due to some error with core. Take a look at the attached error message and retry the submission again.
+3. `TRY_AGAIN_LATER`: this indicates that the stellar-core instance is currently unable to process any submission requests. Clients should wait for sometime before retrying it again.
+
+#### Synchronous Transaction Submission
+
+Due to the blocking nature of the non-async endpoint, things are a little different compared to the async strategy. There are 3 possible scenarios that clients can encounter:
+
+1. The submission is successful and Horizon returns the transaction response back. This is the happy path and clients do not need to do anything but wait for Horizon's response.
+2. Core sends back an `ERROR` response from the submission. Clients should consult the attached error message and retry the submission again.
+3. Timeouts: Horizon may respond with a `504` HTTP code. This response is not an error but a warning that your transaction hasn't been accepted by the network yet. There could be many possible reasons for the timeout, the most common of which is network congestion, but it could also be due to other transient issues. There are a couple things you should to do best handle this response.
+   1. `Polling the transaction hash`: The recommended step is to use the txn hash in the timeout response and poll the `GET /transactions` endpoint to see if it successfully made it into a ledger.
+   2. `Time bounds`: Time bounds are optional but recommended, as they put a time limit on the transaction - so either the transaction makes it onto the ledger or it times out depending on your time parameters.
```

Comment:
This doesn't feel like it belongs as a bullet point of `There are a couple things you should to do best handle this response.`. It's more like a preventative measure. I would probably just rephrase that leading sentence, rather than changing this list though